rblob/blob.go

```diff
@@ -58,16 +57,6 @@ type Option func(*Bucket)
 func OpenBucket(ctx context.Context, label, urlstr string,
 	opts ...Option,
 ) (*Bucket, error) {
-	u, err := url.Parse(urlstr)
```

Comment:
Were there no tests that failed? Perhaps we should add a test case.

Reply:
Thanks for the question. The removed code was for URL parsing and prefix validation which were not explicitly tested, so they continued to pass after this change. The existing tests already cover this function. Could you clarify what type of test case you had in mind?

Reply:
There should have been a test case that passed in a prefix that doesn't end in "/", and the test should have demonstrated that an error was returned.

Now the test case can demonstrate that a prefix that doesn't end in "/" _doesn't_ return an error.

Reply:
As an aside, JP wasn't added to the Luno org, so CI doesn't run by default. I've just approved the CI run, and added you to the org JP 👍

Reply:
Thanks for the suggestion, I have added test cases for prefix with or without slashes.